docs/source/api/apollo-server.md

```diff
@@ -279,6 +279,7 @@ An object containing configuration options for connecting Apollo Server to [Apol
 An array of [plugins](../integrations/plugins) to install in your server instance. Each array element can be either a valid plugin object or a zero-argument function that _returns_ a valid plugin object.
 
 In certain cases, Apollo Server installs some of its built-in plugins automatically (for example, when you provide an Apollo Studio API key with the `APOLLO_KEY` environment variable). For details, see the API references for these plugins: [usage reporting](./plugin/usage-reporting/), [schema reporting](./plugin/schema-reporting/), and [inline trace](./plugin/inline-trace/).
+FIXME AS3: More things are installable plugins now (cache control, landing pages) which should be documented on their own pages and mentioned here.
```

Comment:
I would like to reorganize plugin content and will include this in that work

Reply:
https://github.com/apollographql/apollo-server/pull/5348